### ISSUE TITLE
accessibility compliance in Kendo React Libraries

### ISSUE THREAD

**Issue by IreneMarquet** [telerik/kendo-react#153]
Hello, 

We are going though an accessibility scan in our application, and one of the issues that we have got in the scan is:

<img width="508" alt="captura de pantalla 2018-12-21 a las 14 18 46" src="https://user-images.githubusercontent.com/12745033/50344458-5b990b80-052b-11e9-8c00-188f855ac336.png">

The guidelines provided in this scan are: To help users navigate a page, WAI-ARIA widgets must have an accessible name specified with aria-label, aria-labeledby or the widget's inner text.

Example:  

      <div role="tree" aria-label="table of contents">...</div>

We are using kendo-grid-react-wrapper for our grids, and we've seen that its elements doesn't include: aria-label, where there is a role=. As we can see in the screenshot provided.

I've checked this link: https://www.telerik.com/kendo-react-ui/components/accessibility/ 
but I cant see a way in which I could correct this issue. 

- Could you help me to understand if there is a way we could correct this kind of issue in our application to be accessibility compliant?

Thanks in advance,

**Comment by Xizario** [comment]
I think the issue is that the cells are empty. If there was inner text, the label should not be needed.

The `table` have role `grid` and the `tbody` have role `rowgroup` Even if the w3c suggest the table should have `presentation` or `none` roles, it have been working correctly with `grid` and `tbody` in all readers, and also passing validation in case there is data inside the cells.

Could you confirm if you are using the `kendo-grid-react-wrapper` or the `kendo-react-grid`, because the resolution would be different:
* For the KendoReact Grid you can define cell render or custom cells that can render aria-label if there is no content in the cell. I can prepare you sample for this if needed. 
* And if you using the wrapper of the KendoUI Grid widget, there is probably easy way to select all empty cells with jQuery and apply aria attributes for them. If this is the case, you may also get faster resolution and better help by the [support](https://www.telerik.com/support/kendo-ui) or in the [kendo-ui-core repo](https://github.com/telerik/kendo-ui-core/issues).

Generally speaking readers don't actually need any attributes to read tables correctly. But on a side note we are currently reviewing all components to be (WCAG) 2.1 compatible.

**Comment by simonssspirit** [comment]
This issue will be closed after 7 days is no further help is needed.
